polaris-react/src/components/KeyboardKey/KeyboardKey.scss

```diff
@@ -1,23 +1,32 @@
-$key-base-height: 24px;
+$key-base-height: 28px;
+$medium-side-padding: 9.5px;
 
 .KeyboardKey {
   // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
   display: inline-block;
   // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
   height: $key-base-height;
   margin: 0 var(--p-space-05) var(--p-space-05);
-  padding: 0 var(--p-space-1);
-  background-color: var(--p-surface);
-  box-shadow: 0 0 0 var(--p-space-025) var(--p-border-subdued),
-    0 var(--p-space-05) 0 0 var(--p-surface),
-    0 var(--p-space-05) 0 var(--p-space-025) var(--p-border-subdued);
-  border-radius: var(--p-border-radius-1);
+  padding: 0 $medium-side-padding;
+  background: var(--p-surface-depressed);
+  box-shadow: 0 var(--p-space-025) 0 var(--p-hint-from-direct-light);
+  border-radius: var(--p-space-1);
+  color: var(--p-text);
   font-family: var(--p-font-family-sans);
```

Comment:
This should use the `--p-font-family-mono` token so that we can keep the width same here.